lib/matplotlib/tests/test_tickedstroke.py

```diff
@@ -0,0 +1,70 @@
+from matplotlib.testing.decorators import image_comparison
+import matplotlib.pyplot as plt
+import matplotlib.patheffects as PathEffects
+from matplotlib.path import Path
+import matplotlib.patches as patches
+import numpy as np
+
+
+@image_comparison(['tickedstroke1'], remove_text=True)
```

Comment:
please consider merging the three tests into a single baseline image (e.g. with three subplots) to minimize file size.

Reply:
Done